test/functionalTest/testHarness.sh

```diff
@@ -914,7 +914,8 @@ function testDisabled
   dIx=0
   while [ $dIx -lt  ${#DISABLED[@]} ]
   do
-    if [ test/functionalTest/cases/$testcase == ${DISABLED[$dIx]} ]
+    # Comparison is done based in filename, skipping the path (see https://stackdiary.com/tutorials/bash-get-filename-from-path/)
+    if [ "${testcase##*/}" == "${DISABLED[$dIx]##*/}" ]
```

Comment:
Not actually related with this PR, but found while I was investigating make coverage issues.

NTC (informative)